config/default.yml

```diff
@@ -50,7 +50,7 @@ RSpec/BeforeAfterAll:
   StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/BeforeAfterAll
 
 RSpec/ContextWording:
-  Description: "`context` block descriptions should start with 'when', or 'with'."
+  Description: "`context` block descriptions start with 'when', 'with', or 'without'."
```

Comment:
`Check that` ?

Reply:
The primary reason is:
```
lib/rubocop/cop/rspec/context_wording.rb:6:81: C: Metrics/LineLength: Line is too long. [88/80]
      # Check that `context` block descriptions start with 'when', 'with', or 'without'.
```

If I wrap it, there's another failure:
```

  1) config/default.yml does not have newlines in cop descriptions
     Failure/Error: expect(value).not_to include("\n")
       expected "Check that `context` block descriptions start with 'when', 'with',\nor 'without'." not to include "\n"
     # ./spec/project/default_config_spec.rb:63:in `block (3 levels) in <top (required)>'
```

Anyway, there is a number of cops with descriptions that don't conform to most commonly used 'Check that':

> This cop checks for calling a block within a stub.
> This cop checks void `expect()`.

> Prefer using verifying doubles over normal doubles.

> Checks for stubbed test subjects.

> Enforces use of string to titleize shared examples.

Reply:
Yup this one was also not following it, but at least included `should`, not it just sounds incomplete to me

Reply:
Please take another look, I think I've nailed it.